controlplane/kubeadm/api/v1alpha4/kubeadmcontrolplanetemplate_types.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright The Kubernetes Authors.
```

Comment:
```suggestion
Copyright 2021 The Kubernetes Authors.
```

Usually we add the year when the file was added here (I have no idea if that's mandatory though) (applies to all new files with copyright header)